### PR TITLE
Deleted all unnecessary .WithMessage(...) in fluent validation

### LINKED ISSUE #436: Refactor fluent validation messages
We don't need to write own messages(.WithMessage(".....")) for every checking.

### PR DESCRIPTION
**Pull request by TetianaDomanska-dev**
We don't need to write own messages(.WithMessage(".....")) for every checking.

### CODE REVIEW

File: CharlieBackend.Api/Validators/AccountDTOValidators/ChangeCurrentPasswordDtoValidator.cs [hunk 1]
```diff
@@ -12,17 +12,17 @@ public class ChangeCurrentPasswordDtoValidator : AbstractValidator<ChangeCurrent
...
+                .MaximumLength(30);
             RuleFor(x => x.NewPassword)
-                .NotEmpty().WithMessage("{PropertyName} is required")
-                .MaximumLength(30).WithMessage("Password cannot be greater than {MaxLength} symbols");
+                .NotEmpty()
+                .MaximumLength(30);
             RuleFor(x => x.ConfirmNewPassword)
-                .NotEmpty().WithMessage("{PropertyName} is required")
+                .NotEmpty()
                 .Equal(x => x.NewPassword).WithMessage("Passwords does not match");
```
Comment (andy-dendy-repo) [inline CharlieBackend.Api/Validators/AccountDTOValidators/ChangeCurrentPasswordDtoValidator.cs]:
```suggestion
                .Equal(x => x.NewPassword).WithMessage("Passwords do not match");
```

File: CharlieBackend.Api/Validators/AccountDTOValidators/CreateAccountDtoValidator.cs [hunk 2]
```diff
@@ -11,20 +11,20 @@ public class CreateAccountDtoValidator : AbstractValidator<CreateAccountDto>
...
+                .MaximumLength(50);
             RuleFor(x => x.Password)
-                .NotEmpty().WithMessage("{PropertyName} is required")
-                .MaximumLength(65).WithMessage("Password cannot be greater than {MaxLength} symbols");
+                .NotEmpty()
+                .MaximumLength(65);
             RuleFor(x => x.ConfirmPassword)
-                .NotEmpty().WithMessage("{PropertyName} is required")
+                .NotEmpty()
                 .Equal(x => x.Password).WithMessage("Passwords does not match");
```
Comment (andy-dendy-repo) [inline CharlieBackend.Api/Validators/AccountDTOValidators/CreateAccountDtoValidator.cs]:
```suggestion
                .Equal(x => x.Password).WithMessage("Passwords do not match");
```

**Review by andy-dendy-repo — APPROVED** [review]

**Review by gingerbread-monster — APPROVED** [review]

File: CharlieBackend.Api/Validators/AccountDTOValidators/ChangeCurrentPasswordDtoValidator.cs (hunk 1 above)
Comment (TetianaDomanska-dev) [reply CharlieBackend.Api/Validators/AccountDTOValidators/ChangeCurrentPasswordDtoValidator.cs]: Ok, fixed

File: CharlieBackend.Api/Validators/AccountDTOValidators/CreateAccountDtoValidator.cs (hunk 2 above)
Comment (TetianaDomanska-dev) [reply CharlieBackend.Api/Validators/AccountDTOValidators/CreateAccountDtoValidator.cs]: Ok, fixed

**Review by zelya2021 — APPROVED** [review]

**Review by Serhii-Dudar — APPROVED** [review]

**Pull request merged**